docs/fastddsgen/dataTypes/dataTypes.rst

```diff
@@ -392,16 +462,16 @@ Additionally, the following standard annotations are builtin (recognized and pas
 +-------------------------+--------------------------------------------------------------------------------------------+
 | @autoid                 | [Unimplemented] Automatically allocate identifiers to the elements.                        |
 +-------------------------+--------------------------------------------------------------------------------------------+
-| @optional               | [Unimplemented] Setting an element as optional.                                            |
+| @optional               | Setting an element as optional. More info in `Optional Members`_.                          |
```

Comment:
Maybe it is interesting to define a label so it can be also be referenced outside the scope of this file (for instance the release notes). I suggest creating the label for the optional and extensibility sections.

Reply:
Done in 100644